packages/common/package.json

```diff
@@ -25,6 +25,9 @@
     "winston": "^3.6.0",
     "winston-daily-rotate-file": "^4.6.1"
   },
+  "devDependencies": {
+    "typescript": "^4.6.4"
```

Comment:
DevDependwncies not in single packages please

Reply:
Removed. Just was unhappy, that tsc does not work in specific directory